cicd.md

```diff
@@ -0,0 +1,150 @@
+### Harbor Compose and CI/CD
+
+The `up` command is currently not very CI/CD friendly since it requires login credentials and uses internal-facing APIs.  That's where the `deploy` command comes in.  The `deploy` command can be used to trigger a deployment of new versions of Docker images specified in compose files (one or many shipments with one or many containers).  This works from public build services (e.g.; Circle CI, Codeship, Travis CI, etc.) by using the shipment/environment build token specified using environment variables with the naming convention, `SHIPMENT_ENV_TOKEN`.  
+
+So, for example, to deploy an app with two shipments named, "mss-app-web" and "mss-app-worker" to your dev environment, you would add environment variables to your build.
+
+```
+MSS_APP_WEB_DEV_TOKEN=xyz
+MSS_APP_WORKER_DEV_TOKEN=xyz
+```
+
+And then simply run the following to deploy all containers in all shipments specified in your compose files.
+
+```
+harbor-compose deploy
+```
+
+If you wanted to conditionally deploy to a different environment (e.g., QA) using the same set of compose files, you could...
+
+```
+MSS_APP_WEB_QA_TOKEN=xyz
+MSS_APP_WORKER_QA_TOKEN=xyz
+```
+
+And then simply run
+
+```
+harbor-compose deploy -e qa
```

Comment:
It allows you to deploy to any environment without having to maintain `multiple harbor-compose.yml` files.

Note that there is an issue out there to support variable substitution in harbor-compose.yml but it hasn't been implemented yet.